### DISCUSSION TITLE
model train on colab can not be loaded on the local machine

### DISCUSSION THREAD

**Discussion by phuocblt** [mljar/mljar-supervised#583] [Q&A]
I have one question, why saved model (trained on colab) can not be loaded on local machine?
`supervised.exceptions.AutoMLException: Cannot load AutoML directory. [Errno 2] No such file or directory: '../5_Optuna_RandomForest/framework.json'`

**Comment by pplonski** [comment]
Hi @MossyFighting,

Did you copy all files and directories from AutoML `results_path`? Do you see `5_Optuna_RandomForest/framework.json` file in CoLab?

**Comment by phuocblt** [comment]
Hi @pplonski,
1.  Replied to your questions
    + _Did you copy all files and directories from AutoML results_path?_ Yes, I copied all files into local to do inference
    + _Do you see 5_Optuna_RandomForest/framework.json file in CoLab?_ Yes, I see this file in the local after copying from colab to local
2. I have reproduced the simple example with **Iris dataset**
In this reproducible example, I change mode="Optuna" to mode="Explain" for faster training execution time. 
However, I still can not load the model when copying from Colab (or even EC2 instance) to local machine to do inference by failing load the model. 
You can review the example below.
-----<<<git>>>-----  https://github.com/MossyFighting/mljar_example
Thank you!

**Reply by phuocblt** [reply]
Hi @pplonski I found the error. 
The joblib mismatched between different environments. 
I checked the joblib in Colab and EC2 is 1.2.0, and in my local machine is 1.1.0.
This is reason why I can not load the AutoML model.
Thanks!

**Reply by pplonski** [reply]
Thanks @MossyFighting for finding the issue! I've added a ticket to add `joblib` version when saving the model (#585).
